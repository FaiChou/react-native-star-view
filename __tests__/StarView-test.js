import React from 'React';
import ReactTestRenderer from 'react-test-renderer';
import StarView from '../index';

describe('StarView', () => {
  it('renders correctly', () => {
    const instance = ReactTestRenderer.create(
      <StarView score={3} />
    );
    expect(instance.toJSON()).toMatchSnapshot();
  });
});