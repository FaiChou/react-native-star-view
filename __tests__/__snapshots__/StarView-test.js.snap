// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StarView renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "height": 30,
        "justifyContent": "flex-start",
        "width": 150,
      },
      Object {
        "height": 30,
        "width": 150,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "height": 30,
          "justifyContent": "center",
          "width": 30,
        },
        Object {
          "height": 30,
          "width": 30,
        },
      ]
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../lib/star-yellow.png",
        }
      }
      style={
        Array [
          Object {
            "height": "80%",
            "width": "80%",
          },
          Object {
            "height": 24,
            "width": 24,
          },
        ]
      }
    />
  </View>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "height": 30,
          "justifyContent": "center",
          "width": 30,
        },
        Object {
          "height": 30,
          "width": 30,
        },
      ]
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../lib/star-yellow.png",
        }
      }
      style={
        Array [
          Object {
            "height": "80%",
            "width": "80%",
          },
          Object {
            "height": 24,
            "width": 24,
          },
        ]
      }
    />
  </View>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "height": 30,
          "justifyContent": "center",
          "width": 30,
        },
        Object {
          "height": 30,
          "width": 30,
        },
      ]
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../lib/star-yellow.png",
        }
      }
      style={
        Array [
          Object {
            "height": "80%",
            "width": "80%",
          },
          Object {
            "height": 24,
            "width": 24,
          },
        ]
      }
    />
  </View>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "height": 30,
          "justifyContent": "center",
          "width": 30,
        },
        Object {
          "height": 30,
          "width": 30,
        },
      ]
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../lib/star-gray.png",
        }
      }
      style={
        Array [
          Object {
            "height": "80%",
            "width": "80%",
          },
          Object {
            "height": 24,
            "width": 24,
          },
        ]
      }
    />
  </View>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "height": 30,
          "justifyContent": "center",
          "width": 30,
        },
        Object {
          "height": 30,
          "width": 30,
        },
      ]
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../lib/star-gray.png",
        }
      }
      style={
        Array [
          Object {
            "height": "80%",
            "width": "80%",
          },
          Object {
            "height": 24,
            "width": 24,
          },
        ]
      }
    />
  </View>
</View>
`;
